cmake/thisREST.cmake

```diff
@@ -1,5 +1,17 @@
 # Write thisREST.[c]sh to INSTALL directory
 
+##### Checking if we are using CVMFS LCG GCC11 environment
+set(lcg_found FALSE)
+
+# Check if the CMAKE_COMMAND contains both "cvmfs", "lcg" and "x86_64-el9-gcc11"
```

Comment:
A test like this one (especially the one: x86_64-el9-gcc11 ) will make it more difficult to work if we have to change the release or the OS it is compiled against for whatever reason. Can we try to change it to a test based on the kernel number for instance?

Reply:
This is what I could think of. Not sure about the kernel thing. How the kernel version relates to LCG_104?